test/client_mqtt5.js

```diff
@@ -617,16 +617,16 @@ describe('MQTT 5.0', function () {
 
   it('auth packet', function (done) {
     this.timeout(15000)
-    server.once('client', function (serverClient) {
+    const opts = { host: 'localhost', port: ports.PORTAND115, protocolVersion: 5, properties: { authenticationMethod: 'json' }, authPacket: {} }
+    const client = mqtt.connect(opts)
+    server.once('client', function (c) {
       console.log('server received client')
-      serverClient.on('auth', function (packet) {
+      c.on('auth', function (packet) {
         console.log('serverClient received auth: packet %o', packet)
-        serverClient.end(done)
+        client.end(done)
       })
     })
```

Comment:
@BertKleewein By checking all failed actions seems the only test that occasionally keeps failing was this one. Let's see if this change fixes that too

Reply:
Nope that doesn't fix the test neighter... I have a feel this is caused by a race condition, the auth packet could be sent even before the connack is received and the `c.on('auth')` listener may be set after the auth is received causing the test to fail